nixos/modules/services/networking/lokinet.nix

```diff
@@ -0,0 +1,161 @@
+{ config, lib, pkgs, ... }:
+
+let
+  cfg = config.services.lokinet;
+  settingsFormat = pkgs.formats.ini { listsAsDuplicateKeys = true; };
+  configFile = settingsFormat.generate "lokinet.ini" (lib.filterAttrsRecursive (n: v: v != null) cfg.settings);
+in with lib; {
+  options.services.lokinet = {
+    enable = mkEnableOption "Lokinet daemon.";
+
+    dataDir = mkOption {
```

Comment:
Please remove. `DynamicUser` services aren't compatible with this option so just hard code to `/var/lib/lokinet`.